keystone_spassword/contrib/spassword/mailer.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
New file? Living in the past (2015) ? :)

Reply:
Fixed in: https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/76dd25a8f329e9dc480df3834a6e239c86ca4f4e